src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -463,83 +463,84 @@ public static void configCheck(final ConfigType type) {
 			earthBlocks.add("#emerald_ores"); // added in 1.17
 			earthBlocks.add("#lapis_ores"); // added in 1.17
 			earthBlocks.add("#redstone_ores"); // added in 1.17
-			earthBlocks.add("CALCITE"); // use Material in 1.17
-			earthBlocks.add("DRIPSTONE_BLOCK"); // use Material in 1.17
-			earthBlocks.add("LARGE_AMETHYST_BUD"); // use Material in 1.17
-			earthBlocks.add("MEDIUM_AMETHYST_BUD"); // use Material in 1.17
-			earthBlocks.add("SMALL_AMETHYST_BUD"); // use Material in 1.17
-			earthBlocks.add(Material.ANCIENT_DEBRIS.toString());
-			earthBlocks.add(Material.CLAY.toString());
-			earthBlocks.add(Material.COAL_ORE.toString()); // no longer needed in 1.17
-			earthBlocks.add(Material.COARSE_DIRT.toString());
-			earthBlocks.add(Material.COBBLESTONE.toString());
-			earthBlocks.add(Material.COBBLESTONE_SLAB.toString());
-			earthBlocks.add(Material.DIAMOND_ORE.toString()); // no longer needed in 1.17
-			earthBlocks.add(Material.DIRT.toString());
-			earthBlocks.add(Material.EMERALD_ORE.toString()); // no longer needed in 1.17
-			earthBlocks.add(Material.GRASS_BLOCK.toString());
-			earthBlocks.add(Material.GRASS_PATH.toString());
-			earthBlocks.add(Material.GRAVEL.toString());
-			earthBlocks.add(Material.LAPIS_ORE.toString()); // no longer needed in 1.17
-			earthBlocks.add(Material.MYCELIUM.toString());
-			earthBlocks.add(Material.PODZOL.toString());
-			earthBlocks.add(Material.REDSTONE_ORE.toString()); // no longer needed in 1.17
-			earthBlocks.add(Material.STONE_SLAB.toString());
+			earthBlocks.add("CALCITE");

```

Comment:
You may want to compare this list of added earth blocks with @Simplicitee because he has a similar list in his pr #1188. You have calcite and he doesn't for starters.

Reply:
The list should probably be in his PR anyway because it's the main 1.18 PR

Reply:
It's not the main 1.18 PR because it doesn't support 1.16-1.18.